en/components/skeleton-charts.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Skeleton Charts - Design System Component
+_description: The Skeleton Charts are representations of the various Chart components that are shown while data is being loaded in the background to provide content for its series.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+# Skeleton Charts
+
+Use the Skeleton Charts to display the same type of information that you would in the standard ones, but to also indicate an application state when no data is present at the moment e.g. when series and data values are loaded from a remote data source and there is either lack of connectivity, or it takes some time. The Skeleton Charts are just a visual representation of a Chart with a known type, but without any data to show.
+
+## Skeleton Chart Demo
+
+<img class="responsive-img" src="../images/skeleton_charts_column.png" srcset="../images/skeleton_charts_column@2x.png 2x" />
+
+## Charts
+
+Besides the Column Chart, we also have Skeleton variants for Line, Spline, and Step Line. 
+
+<img class="responsive-img" src="../images/skeleton_charts_line.png" srcset="../images/skeleton_charts_line@2x.png 2x" />
+<img class="responsive-img" src="../images/skeleton_charts_spline.png" srcset="../images/skeleton_charts_spline@2x.png 2x" />
+<img class="responsive-img" src="../images/skeleton_charts_step.png" srcset="../images/skeleton_charts_step@2x.png 2x" />
+
+## Title
+
+The title is visible and shows the name of the Skeleton Chart because unlike data it should always be instantly available.
+
+<img class="responsive-img" src="../images/skeleton_charts_title.png" srcset="../images/skeleton_charts_title@2x.png 2x" />
+
+## Legend
+
+Skeleton Charts come with a Legend which is also in Skeleton state since the amount of series may not be available at first.
+
+<img class="responsive-img" src="../images/skeleton_charts_legend.png" srcset="../images/skeleton_charts_legend@2x.png 2x" />
+
+## Chart Type
+
+The Skeleton Line, Spline, and Step Charts come with the option to show area instead of a line, similarly to their databound counterparts.
```

Comment:
```suggestion
The Skeleton Line, Spline, and Step Charts come with the option to show area instead of a line, similarly to their data-bound counterparts.
```